util/secret/certificates.go

```diff
@@ -65,33 +65,43 @@ type Certificates []*Certificate
 
 // NewCertificatesForInitialControlPlane returns a list of certificates configured for a control plane node.
 func NewCertificatesForInitialControlPlane(config *bootstrapv1.ClusterConfiguration) Certificates {
```

Comment:
Can we have a test that NewCertificatesForInitialControlPlane with a custom  CACertificateValidityPeriodDays has the desired effect (and other around, if CACertificateValidityPeriodDays, 10y is used)